### ISSUE TITLE
&minus; code not converted to minus symbol as expected

### ISSUE THREAD

**Issue by orware** [metabase/metabase#7123]
### Bugs

This is just something minor I noticed right now and shows up when creating a new question (I click on "..." on the far right and then click on the "Custom Fields" option.

I searched the repository and it looks like this line of code is the one having the issue:
https://github.com/metabase/metabase/blob/master/frontend/src/metabase/query_builder/components/expressions/ExpressionWidget.jsx#L57

Basically, you guys have a + symbol directly in the string, but for the - symbol you've opted to use the &minus; code but it's not being converted on display correctly so you guys can choose what'll be the best way to fix the issue, but just wanted to report it so you're aware!
